tests/unit_test.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+import unittest
+
+from gameyamlspiderandgenerator.util.spider import get_status, get_text
+
+
+def update_config():
+    from gameyamlspiderandgenerator.util.setting import config, set_config
+
+    config(
+        {
+            "proxy": {
+                "http": "http://127.0.0.1:10809",
+                "https": "http://127.0.0.1:10809",
+            },
+            "api": {
+                "google-play": "a714b00383f0662a61b2e382d55c685f17015617aa7048972da58a756fb75e90",
+                "apple": "a714b00383f0662a61b2e382d55c685f17015617aa7048972da58a756fb75e90",
+            },
+        }
```

Comment:
```suggestion
        {
            "proxy": {"http": ..., "https": ...},
            "api": {
                "google-play": ...,
                "apple": ...,
            },
        }
```

1. Omit config for `proxy` and `api`, move to somewhere else, or read from a config file